### DISCUSSION TITLE
Does it support field name transform before response to client?

### DISCUSSION THREAD

**Discussion by RomanMIzulin** [collerek/ormar#823] [Q&A] [answered]
For example, database model has ` product_field_name1` , but an application supposed to transfer value to client under `super_product_fieldname`.

**Comment by collerek** [comment, marked as answer]
Yes it does, you can set different model field name and different database name with name: https://collerek.github.io/ormar/fields/common-parameters/#name
You can also utilize property fields: https://collerek.github.io/ormar/models/#property-fields
